packages/core/src/button/register.ts

```diff
@@ -4,17 +4,18 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import '@clr/core/badge/register.js';
```

Comment:
Do we not need to import badges anymore?

Reply:
We apply styles if projected into a slot but its not actually used in the JS of the button. Since its not we can omit it and it wont get bundled into the button regardless if you used it or not.